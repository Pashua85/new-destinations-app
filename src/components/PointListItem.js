import React from 'react';

const PointListItem = (props) => (
  <div className="point-list__item">
    <div className="point-list__title">
      {props.item.pointName}
    </div>
    <button className='point-list__btn remove-btn'>Delete</button>
  </div>
);

export default PointListItem;