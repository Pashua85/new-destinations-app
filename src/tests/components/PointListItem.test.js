import React from 'react';
import { shallow } from 'enzyme';
import PointListItem from '../../components/PointListItem';
import points from '../utils/points';

test('should render PointListItem with static data', () => {
  const wrapper = shallow(<PointListItem item={points[0]} />);
  expect(wrapper).toMatchSnapshot();
});