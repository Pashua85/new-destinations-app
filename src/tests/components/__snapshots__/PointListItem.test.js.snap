// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PointListItem with static data 1`] = `
<div
  className="point-list__item"
>
  <div
    className="point-list__title"
  >
    Point one
  </div>
  <button
    className="point-list__btn remove-btn"
  >
    Delete
  </button>
</div>
`;
